src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/SymmetricKeyWrap.cs

```diff
@@ -104,42 +125,50 @@ internal static byte[] AESKeyWrapEncrypt(byte[] rgbKey, byte[] rgbWrappedKeyData
             rijn.Mode = CipherMode.ECB;
             rijn.Padding = PaddingMode.None;
```

Comment:
I'm normally not a fan of making superfluous changes, but we're already fixing the disposing pattern here, so may as well go to town.

Would you mind, terribly, changing this from RijndaelManaged to Aes (via Aes.Create())?

It's just so much more clear what's going on when it says Aes instead of Rijndael.  Especially in a method with "AES" in its name.